### PR TITLE
add sd1.5 test_overfit vanilla 8p

### PR DESCRIPTION
**Pull request by Songyuanwei**

### CODE REVIEW

File: examples/stable_diffusion_v2/tests/st/test_overfit.py [hunk 1]
```diff
@@ -38,6 +42,70 @@ def create_dataset(n=1):
...
+    else:
+        raise ValueError(f"SD {version} not included in test")
+
+    output_path = root
+    output_path = os.path.join(output_path, "vanilla_8p")
+
+    os.makedirs(output_path, exist_ok=True)
+
+    # export MS_ASCEND_CHECK_OVERFLOW_MODE="INFNAN_MODE"
+    os.environ["MS_ASCEND_CHECK_OVERFLOW_MODE"] = "INFNAN_MODE"  # It depends on MS version
```
Comment (wtomin) [inline examples/stable_diffusion_v2/tests/st/test_overfit.py:76]: What ms version it is dependent on? You may set it differently for different ms versions.

Comment (Songyuanwei) [reply examples/stable_diffusion_v2/tests/st/test_overfit.py:76]: 这是在ms2.2.10上测试用的测试用例，之前的都加了，为了以防万一，这个就没有取消

Comment (wtomin) [reply examples/stable_diffusion_v2/tests/st/test_overfit.py:76]: Got it.

**Review by wtomin — APPROVED** [review]

File: examples/stable_diffusion_v2/tests/st/test_overfit.py (hunk 1 above)
Comment (Songyuanwei) [reply examples/stable_diffusion_v2/tests/st/test_overfit.py:76]: 不加的话，会overflow， loss不会下降

**Review by SamitHuang — APPROVED** [review]

File: examples/stable_diffusion_v2/tests/st/test_overfit.py [hunk 2]
```diff
@@ -1,7 +1,11 @@
 """
+vanilla_8p, unet randomly initialized, loss change 1.0 ->  0.3
 Vanilla, unet randomly initialized, loss change 1.0 ->  0.1
 LoRA, loss not stable, but 0.5 is a safe threshold
 Dreambooth, unet randomly initialized, loss  change 2.0 -> 0.5
+Before running the `test_vanilla_8p`, please ensure that the `chinese_art_blip` dataset
+is located in the `path_to_examples/stable_diffusion_v2/datasets/` folder.
+The 'chinese_art_blip' dataset url is "https://openi.pcl.ac.cn/attachments/c1941496-fafc-4074-be7b-75fa9f803a53?type=1"
 """
```
Comment (SamitHuang) [inline examples/stable_diffusion_v2/tests/st/test_overfit.py:9]: Put these running instruction to a readme file under `tests/st/get_tested.md`

Comment (Songyuanwei) [reply examples/stable_diffusion_v2/tests/st/test_overfit.py]: done

**Review by SamitHuang — APPROVED** [review]

**Pull request merged**